testing/py-webencodings/APKBUILD

```diff
@@ -0,0 +1,51 @@
+# Contributor: Stefan Wagner <stw@bit-strickerei.de>
+# Maintainer: Stefan Wagner <stw@bit-strickerei.de>
+pkgname=py-webencodings
+_pkgname=webencodings
+pkgver=0.5.1
+pkgrel=0
+pkgdesc="Character encoding for the web"
+url="https://github.com/gsnedders/python-webencodings"
+arch="noarch"
+license="MIT"
+makedepends="python2-dev python3-dev py-setuptools"
+subpackages="py2-$_pkgname:_py2 py3-$_pkgname:_py3"
+source="$_pkgname-$pkgver.tar.gz::https://github.com/gsnedders/python-$_pkgname/archive/v$pkgver.tar.gz"
+builddir="$srcdir"/python-$_pkgname-$pkgver
+
+build() {
+	cd "$builddir"
+	python2 setup.py build
+	python3 setup.py build
+}
+
+check() {
+	python2 setup.py test
```

Comment:
Needs add `cd "$builddir"`

Reply:
I had something in mind with newest abuild changes dirs to $builddir automatically.
Added.